### DISCUSSION TITLE
Import "awswrangler" could not be resolved Pylance reportMissingImports

### DISCUSSION THREAD

**Discussion by debbie-israel** [microsoft/pyright#4362] [General]
Cannot use awswrangler library in VSCode.
The following error is given:

![image](https://user-images.githubusercontent.com/56561804/208695298-283dc685-1ad4-4d68-9d44-7906885b7649.png)

Import "awswrangler" could not be resolvedPylance[reportMissingImports](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportMissingImports)

Thanks in advance

**Comment by erictraut** [comment]
Transferring this to a discussion topic since it's very unlikely a bug.

**Reply by erictraut** [reply]
You haven't provided any repro steps, version information, etc. That leaves me with insufficient information to help you diagnose the problem. The most common reason for an error like this is a configuration problem where you haven't told pyright which Python environment you want it to use. For documentation, refer to [this documentation](https://github.com/Microsoft/pyright/blob/main/docs/import-resolution.md). If you still can't figure out the problem, feel free to provide more information here, and I'll try to help you diagnose.
